packages/vendure-plugin-customer-managed-groups/src/customer-managed-groups.service.ts

```diff
@@ -364,4 +371,141 @@ export class CustomerManagedGroupsService {
       isGroupAdministrator,
     };
   }
+
+  async updateGroupMember(
+    ctx: RequestContext,
+    input: UpdateCustomerManagedGroupMemberInput
+  ): Promise<CustomerManagedGroup> {
+    if (
+      !input.title &&
+      !input.firstName &&
+      !input.lastName &&
+      !input.emailAddress
+    ) {
+      throw new UserInputError(`empty body`);
+    }
+    const myGroup = await this.myCustomerManagedGroup(ctx);
+    if (myGroup) {
+      const groupAdmin = myGroup.customers.find((c) => c.isGroupAdministrator);
+      if (!groupAdmin) {
+        throw new UserInputError('group has no admin');
+      }
+      const groupAdminCustomer = await this.customerService.findOne(
+        ctx,
+        groupAdmin?.customerId,
+        ['user']
+      );
+      for (let member of myGroup.customers) {
+        if (member.customerId === input.customerId) {
+          const userRepo = getRepository(User);
+          const customer = await this.customerService.findOne(
+            ctx,
+            member.customerId,
+            ['user']
+          );
+          if (!customer || !customer.user) {
+            throw new UserInputError(
+              `no customer with id ${input.customerId} exists`
```

Comment:
Annoying, but cleaner: Please make sure Errors and log statements are capitalized
```suggestion
              `No customer with id ${input.customerId} exists`
```